code/modules/mob/living/carbon/human/appearance.dm

```diff
@@ -235,6 +235,14 @@
 		update_body()
 	return TRUE
 
+/mob/living/carbon/human/proc/change_runechat_color(colour = "#000000", update_dna = TRUE, skip_icons = TRUE)
```

Comment:
when would you ever want `update_dna ` to be false? I dont think theres any proc instances here where its false

Reply:
Never, this was an oversight from me taking reference of the proc above it, `change_eye_color`